gpkit/tools/tools.py

```diff
@@ -113,3 +115,63 @@ def mdmake(filename, make_tex=True):
         with open(filename+".tex.md", "w") as f:
             f.write(texmd)
     return open(filename+".py")
+
+
+def bound_all_variables(varkeys, lb, ub):
+    "Returns model with additional constraints bounding all free variables"
```

Comment:
looks like this returns a list of constraints, not a Model. Also I think the name could be better -- `bounded_variable_constraints` or `bounded_variables` or `get_bounds`?